### PR TITLE
Organization UI

### PR DESCRIPTION
**Pull request by imaximumGit**
The basic design of the organization UI is implemented and a global store for the app is implemented (redux).

### CODE REVIEW

File: PopApp/Components/Witness.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  text: {
+    ...Typography.base,
+  },
+  button: {
+    ...Buttons.base,
+  },
+});
+
+const Witness = ({ navigation }) => (
+  <FlatList
```
Comment (RaulinN) [inline PopApp/Components/Witness.js:27]: Both `data` and `renderItem` attributes must be specified in a `FlatList` according to the [documentation](https://reactnative.dev/docs/flatlist)

**Review by RaulinN — CHANGES_REQUESTED** [review]
Great job @imaximumGit ! Everything seems to work well! :D 

ℹ️ I didn't review code related to data storage since it is temporary for now. I will add the persistent storage once this PR gets accepted. I also made sure that your current PR compiles/works with the new storage :)

**Review by RaulinN — APPROVED** [review]
👍

**Review by lhmerino — CHANGES_REQUESTED** [review]
Looks good but please add more descriptive comments. For example: Instead of "Manage the Home screen" have it say: "Manages the home screen components: a Welcome string, a LAO connect button, and a launch LAO button." And if possible, the link between components such as "Launch LAO" button is linked to the Launch component.

File: PopApp/Components/DiscussionEvent.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {
+  StyleSheet, View, Text, FlatList, TextInput, Button,
+} from 'react-native';
+
+import EventItem from './EventItem';
+import { Spacing } from '../Styles';
+import PROPS_TYPE from '../res/Props';
+
+const QUESTION = [
```
[the remark this answers is not in the document]
Comment (imaximumGit) [reply PopApp/Components/DiscussionEvent.js:11]: It is some fictive questions/discussions to show how the discussion event component works.

**Review by lhmerino — APPROVED** [review]

**Pull request merged**